mito-sql-cell/src/magicLineUtils.ts

```diff
@@ -0,0 +1,160 @@
+import type { ICodeCellModel } from '@jupyterlab/cells';
+
+/**
+ * The jupysql configuration needed for Mito SQL.
```

Comment:
No longer jupysql

Reply:
Actually, that's not true, we still rely on jupysql for syntax highlighting for now.